### PR TITLE
Add FAQ about decision to use Postgres protocol

### LINKED ISSUE #1839: FAQ: Why use the postgres protocol instead of mysql?
I've answered this a lot, probably time to add it to the FAQ: https://forum.cockroachlabs.com/t/why-cockroachdb-not-support-mysql-protocol/895/2

### PR DESCRIPTION
**Pull request by jseldess**
Fixes #1839

### CODE REVIEW

File: v1.0/frequently-asked-questions.md [hunk 1]
```diff
@@ -125,6 +125,12 @@ CockroachDB supports the PostgreSQL wire protocol, so you can use any available
 
 See [Install Client Drivers](install-client-drivers.html) for more details.
 
+## Why does CockroachDB use the PostgreSQL wire protocol instead of the MySQL protocol?
+
+CockroachDB uses the PostgreSQL wire protocol because it is better documented than the MySQL protocol and the PostgreSQL license is better for us than the MySQL license.
```
Comment (petermattis) [inline v1.0/frequently-asked-questions.md]: PostgreSQL uses the PostgreSQL license: "a liberal Open Source license, similar to the BSD or MIT licenses."

MySQL is licensed under the GPL which is much more restrictive. @bdarnell can provide better words here.

Comment (jseldess) [inline v1.0/frequently-asked-questions.md]: Updated using more of your language, in the meantime.

**Comment by bdarnell** [comment]
:lgtm:

---

Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/1845#-:-KrumGBGNFt1IquZeR6Q:bnfp4nl)*

**Comment by tamird** [comment]
@jseldess assuming the bullet about better documentation is referencing our dinner conversation, I believe the original reason was that postgres' *dialect* was better documented than mysql's.

Not sure this needs fixing, but wanted to clarify, in case I was the cause of misconception.

**Comment by bdarnell** [comment]
This came from an answer I gave on the forum, and I was referring to the protocol being better documented.

**Comment by jseldess** [comment]
Haha, meant to mention at dinner that I'd added that FAQ. Thanks for finding it, @tamird.

**Pull request merged**